cmd/libs/go2idl/deepcopy-gen/generators/deepcopy.go

```diff
@@ -0,0 +1,388 @@
+/*
```

Comment:
Yes - that's what I thought about.
I will try to send out a separate PR for it (if I manage to do it today).